grammars/julia.cson

```diff
@@ -218,11 +218,15 @@ repository:
         name: "keyword.operator.arrow.julia"
       }
       {
-        match: "(?:\\.==|!=|\\.>=|\\.>=|\\.>|\\.<|==|\\.!=|\\.=|\\.!|<:|:>|>=|<=|>|<)"
+        match: "(?:<<(?!=)|>>>(?!=)|>>(?!=|>=)|\\.>>>|\\.>>|\\.<<)"
+        name: "keyword.operator.shift.julia"
+      }
+      {
+        match: "(?:===|\\.==|!==|!=|\\.>=|\\.>(?!=)|\\.<(?!=)|==|\\.!=|\\.=|\\.!|<:|:>|(?<!>)>=|(?<!<)<=|\\.<=|>(?!>)|<(?!<))"
```

Comment:
I'm pretty sure the lookaheads for `\\.>(?!=)` and `\\.<(?!=)` shouldn't be necessary in the first place - were ligatures not working for those beforehand?  The others are unnecessary if `keyword.operator.update` is placed above these.